nodes/scheduling/secondary_scheduler/nodes-secondary-scheduler-release-notes.adoc

```diff
@@ -10,7 +10,30 @@ The {secondary-scheduler-operator-full} allows you to deploy a custom secondary
 
 These release notes track the development of the {secondary-scheduler-operator-full}.
 
-:operator-name: The {secondary-scheduler-operator}
-include::snippets/operator-not-available.adoc[]
-
 For more information, see xref:../../../nodes/scheduling/secondary_scheduler/index.adoc#nodes-secondary-scheduler-about_nodes-secondary-scheduler-about[About the {secondary-scheduler-operator}].
+
+[id="secondary-scheduler-operator-release-notes-1.4.0_{context}"]
+== Release notes for {secondary-scheduler-operator-full} 1.4.0
+
+Issued: 9 April 2025
+
+The following advisory is available for the {secondary-scheduler-operator-full} 1.4.0:
+
+* link:https://access.redhat.com/errata/RH??-2025:TODO[RH??-2025:TODO]
+
+// TODO: Update errata link once available
+
+[id="secondary-scheduler-1.4.0-new-features_{context}"]
+=== New features and enhancements
+
+* This release of the {secondary-scheduler-operator} updates the Kubernetes version to 1.31.
```

Comment:
@ingvagabund @zhouying7780 I added this line because we had started noting when these operator versions bumped the kube version. Let me know if this looks okay to you!

Reply:
This looks great. Simple and informative.